Sources/CreateAPI/Generator/Generator+Package.swift

```diff
@@ -8,19 +9,19 @@ extension Generator {
     }
 
     func makePackageFile(name: String) -> String {
-        let packages: String = [
-            #".package(url: "https://github.com/kean/Get", from: "1.0.2")"#,
-            isHTTPHeadersDependencyNeeded ? #".package(url: "https://github.com/CreateAPI/HTTPHeaders", from: "0.1.0")"# : nil,
-            isNaiveDateNeeded ? #".package(url: "https://github.com/CreateAPI/NaiveDate", from: "1.0.0")"# : nil,
-            isQueryEncoderNeeded ? #".package(url: "https://github.com/CreateAPI/URLQueryEncoder", from: "0.2.0")"# : nil
-        ].compactMap { $0 }.joined(separator: ", \n")
+        let allPackages = options.package.dependencies
+            .appending(GeneratorPackages.Get)
+            .appending(if: isHTTPHeadersDependencyNeeded, GeneratorPackages.HTTPHeaders)
+            .appending(if: isNaiveDateNeeded, GeneratorPackages.NaiveDate)
+            .appending(if: isQueryEncoderNeeded, GeneratorPackages.URLQueryEncoder)
```

Comment:
This is so much better 😍 

A couple of suggestions:

1. Define define the constants on `PackageDeclaration` so that we can drop the repetition
2. Constants should still use the `lowerCamelCase` syntax 
3. For the function to read a little more naturally, maybe `if` could be the second argument? 

For example:

```suggestion
        let allPackages = options.package.dependencies
            .appending(.get)
            .appending(.httpHeaders, if: isHTTPHeadersDependencyNeeded)
            .appending(.naiveDate, if: isNaiveDateNeeded)
            .appending(.urlQueryEncoder, if: isQueryEncoderNeeded)
```